util/tikvutil/tikvutil.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tikvutil
+
+import (
+	"go.uber.org/atomic"
+)
+
+// CommitterConcurrency stores the current value of the sysvar tidb_committer_concurrency
+var CommitterConcurrency = atomic.NewInt32(128)
```

Comment:
Note to other reviewers: this package was created to break the import cycle of config requiring variable package, and variable requiring config. The default value of 128 will be overwritten as soon as the sysvar cache is loaded, which happens during startup.